mlc_config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "ignorePatterns": [
+        {"pattern": "http://127.0.0.1:8000/"},
+        {"pattern": "^https://jsonformatter.curiousconcept.com/"},
+        {"pattern": "^http.*://schema.repronim.org/.*"},
+        {"pattern": "^http://schema.org/version/2.0/"}
```

Comment:
FYI this invalid link is only used as an example

https://github.com/ReproNim/reproschema/blob/f1d4c2bdee57bdc92fb911b83bbc3a2288e746d8/docs/30_schema.md?plain=1#L79

Reply:
looks good! (sorry, i didn't get any notifications of this....